app/views/Project/EntryEdit/index.tsx

```diff
@@ -311,6 +365,25 @@ function EntryEdit(props: Props) {
                     );
                 }
 
+                // eslint-disable-next-line max-len
```

Comment:
Refrain from disabling this rule when we can just break our expressions in multiple line